wordle-web/src/views/EditorView.vue

```diff
@@ -0,0 +1,164 @@
+    <v-list scrollable height="300px" width="100px">
+      <v-list-item
+        v-for="(item, i) in validWords(add)"
+        :key="i"
+        :value="item"
+        active-color="primary"
+        variant="plain"
+        :title="item"
+        @click="fillAdd(item)"
+      />
+    </v-list>
+
+    <v-checkbox v-model="checkbox1" :label="'Is A Common Word?'"></v-checkbox>
+    <v-btn @click="addWord"> Add Word </v-btn>
+    <h3>{{ addError }}</h3>
+  </v-container>
+
+  <v-container>
+    <h2>Update IsCommon</h2>
+    <v-text-field v-model="update" label="Update Word" variant="outlined"></v-text-field>
+    <v-list scrollable height="300px" width="100px">
+      <v-list-item
+        v-for="(item, i) in validWords(update)"
+        :key="i"
+        :value="item"
+        active-color="primary"
+        variant="plain"
+        :title="item"
+        @click="fillUpdate(item)"
+      />
+    </v-list>
+
+    <v-checkbox v-model="checkbox2" :label="'Is A Common Word?'"></v-checkbox>
+    <v-btn @click="updateWord"> Update Word </v-btn>
+    <h3>{{ updateError }}</h3>
+  </v-container>
+  <v-container>
+    <h2>Delete Word</h2>
+    <v-text-field v-model="remove" label="Delete Word" variant="outlined"></v-text-field>
+    <v-list scrollable height="300px" width="100px">
+      <v-list-item
+        v-for="(item, i) in validWords(remove)"
+        :key="i"
+        :value="item"
+        active-color="primary"
+        variant="plain"
+        :title="item"
+        @click="fillDelete(item)"
+      />
+    </v-list>
+    <v-btn @click="removeWord"> Delete Word </v-btn>
+    <h3>{{ deleteError }}</h3>
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import { EditorService } from '@/scripts/editorService'
+import Axios from 'axios'
+import { onMounted, ref } from 'vue'
+
+const allWords = ref(['', ''])
+const allIsCommon = ref(['', ''])
+
+const addError = ref('')
+const add = ref('')
+const checkbox1 = ref(false)
+const updateError = ref('')
+const update = ref('')
+const checkbox2 = ref(false)
+const deleteError = ref('')
+const remove = ref('')
+
+onMounted(async () => {
+  allWords.value = await EditorService.getAllWords()
+  allIsCommon.value = await EditorService.getAllIsCommon()
+})
+
+async function addWord() {
+  let newWord = add.value.toLocaleLowerCase()
+  if (newWord.length != 5) {
+    addError.value = 'Word length must be 5 characters.'
+  } else if (allWords.value.includes(newWord)) {
+    addError.value = 'Word already exists.'
+  } else {
+    Axios.post('/Word/AddWord?word=' + newWord + '&isCommon=' + checkbox1.value)
+      .then(function (response) {
+        console.log(response)
+      })
+      .catch(function (error) {
+        console.log(error)
+      })
+    addError.value = 'New Word Added: ' + newWord + ', Is Common: ' + checkbox1.value
+    allWords.value = await EditorService.getAllWords()
+    allIsCommon.value = await EditorService.getAllIsCommon()
```

Comment:
You should avoid returning all words to the user at all costs because this takes forever! Paginate on the backend and only send what you need! Take a look that what he did in class, it should help you out a lot and let me know if you have any questions.

Reply:
Noted! I'll look into implementing that!